### PR TITLE
Feature 17

### LINKED ISSUE #17: Push Notifications - Notify when out of item
- [x] Build Push Notification Manager singleton
- [x] Integrate Push notifs into decrement operation

### PR DESCRIPTION
**Pull request by isaacd9**
Resolves #17

### CODE REVIEW

**Comment by isaacd9** [comment]
Ping for review @Satre95

File: app/src/main/java/com/ucsd/cse110t15m260/roommateinventory/InventoryCellAdapter.java [hunk 1]
```diff
@@ -5,7 +5,10 @@
  */
 
 import android.content.Context;
+<<<<<<< HEAD
```
Comment (isaacd9) [inline app/src/main/java/com/ucsd/cse110t15m260/roommateinventory/InventoryCellAdapter.java]: Whoops. Let me resolve

**Comment by 0xTenable** [comment]
Looks good besides the spelling :+1:

**Comment by isaacd9** [comment]
@LeoWW Mergin'.

**Pull request merged**